README.md

```diff
@@ -51,14 +51,53 @@ See [smbus2](https://pypi.org/project/smbus2/) for more information.
 ### USB Interface
 
 Requires `hid` to be installed and accessible to your Python interpreter.
-You will usually need sudo access for this, so:
 
 ```console
-sudo python3 -m pip install hid
+pip install hid==1.0.4
 ```
 
 See [hid](https://pypi.org/project/hid/) for more information.
 
+#### Linux
+
+Using the `hid` package will access the TouchNetix protocol bridges via the `/dev/hidrawX` interface. This typically requires root access. This means the scripts will need to be run as `sudo`. Alternativly, `udev` can be used to give all users permissions to the `hidraw` devices.
+
+Create the following `udev` rules file `/etc/udev/rules.d/99-axiom-hidraw-permissions.rules`:
+
+```text
+SUBSYSTEM=="hidraw", ATTRS{idVendor}=="03eb", ATTRS{idProduct}=="6f02", MODE="0666"
+SUBSYSTEM=="hidraw", ATTRS{idVendor}=="03eb", ATTRS{idProduct}=="2f04", MODE="0666"
+SUBSYSTEM=="hidraw", ATTRS{idVendor}=="03eb", ATTRS{idProduct}=="2f08", MODE="0666"
+
+SUBSYSTEM=="hidraw", ATTRS{idVendor}=="0483", ATTRS{idProduct}=="6f02", MODE="0666"
+SUBSYSTEM=="hidraw", ATTRS{idVendor}=="0483", ATTRS{idProduct}=="2f04", MODE="0666"
+SUBSYSTEM=="hidraw", ATTRS{idVendor}=="0483", ATTRS{idProduct}=="2f08", MODE="0666"
+
+SUBSYSTEM=="hidraw", ATTRS{idVendor}=="28e9", ATTRS{idProduct}=="6f02", MODE="0666"
+SUBSYSTEM=="hidraw", ATTRS{idVendor}=="28e9", ATTRS{idProduct}=="2f04", MODE="0666"
+SUBSYSTEM=="hidraw", ATTRS{idVendor}=="28e9", ATTRS{idProduct}=="2f08", MODE="0666"
+```
+
+The changes will apply on the next bootup. To apply the changes immediatly:
```

Comment:
Typo: "immediately"

Reply:
There were a number of typos in the README.md file, I have installed a spell checker and have fixed the others also.